icons/lapce.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Lapce</title><path d="M1.607 0v24l6.394-3.691v-2.534L3.802 20.2V1.267Zm4.2 2.424v14.014l2.203-1.27V6.229c3.33 1.924 6.662 3.846 9.993 5.77l-7.799 4.502v2.535l9.992-5.77 2.196-1.268-2.195-1.268L8.01 3.697 8 3.691zm4.397 7.546v3.93l3.402-1.965z"/></svg>
```

Comment:
The original SVG file as well as this converted one leave some room for optimization:

- There is a redundant path point right before the closing `Z` of each path segment which can be removed because it's implicitly given by the `Z`.
- There are redundant path points on straight lines which can be removed:
  ![lapce-svg-points](https://user-images.githubusercontent.com/2206639/165553171-b48b5094-32b0-4caf-ac7b-542562d0f6d5.png)

- Cubic path segments are sometimes used for straight lines which are inefficient and not needed.

I have derived the icon from the original one again and optimized it which leads to a reduction in the path data by ~30%:

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Lapce</title><path d="M3.802 1.267 1.608 0v24L8 20.31v-2.535L3.802 20.2Zm4.208 13.9V6.231L18.003 12l-7.798 4.503v2.533L22.392 12 5.806 2.424V16.44Zm5.598-3.231L10.205 9.97v3.93Z"/></svg>
```